getting-started/templates/systemlink-secrets.yaml

```diff
@@ -390,7 +390,7 @@ saltmaster:
 ##
 systems:
   secrets:
-    ## Cryptographic key to be used for AES-128 encryption of data. This key should have a length of 32 bytes.
+    ## Cryptographic key to be used for AES-256 encryption of data. This key should have a length of 32 bytes.
```

Comment:
How about this (applies to the other string as well)?
- 'This AES-256 key encrypts session data. Ensure the key is 32 bytes long.'